apps/music/js/ui/views/player_view.js

```diff
@@ -788,7 +783,8 @@ var PlayerView = {
     this.seekSlider.setAttribute('aria-valuenow', currentTime);
 
     // if endTime is 0, that's a reset of seekBar
-    var ratio = (endTime !== 0) ? (currentTime / endTime) : 0;
+    var ratio = (isFinite(endTime) && endTime !== 0) ?
+        (currentTime / endTime) : 0;
```

Comment:
I think this change effectively revert the workaround from 1150322. If gecko gives us an infinte duration for some reason (like if we ever add support for audio streams) then we will have the symptoms from 1150322 again. But 1150322 was resolved with a gecko fix, so maybe that is okay.